app/views/admin/editions/_style_guidance.html.erb

```diff
@@ -0,0 +1,61 @@
+<h2 class="govuk-heading-l">Styling</h2>
+
+<p class="govuk-body">For styles, see the <%= link_to "style guide", "https://www.gov.uk/guidance/style-guide", class: "govuk-link" %>
```

Comment:
Is this supposed to be here? I think it effectively repeats the `<p>` below it.

<img width="290" alt="Cursor_and_Editing_publication_HTML_attachment_-_GOV_UK_Whitehall_Publisher" src="https://user-images.githubusercontent.com/7735945/193307811-5a146fad-de54-4c2d-abe5-dde33546e7ac.png">